src/qibolab/instruments/qm.py

```diff
@@ -65,15 +68,13 @@ def declare_output(self, threshold=None, iq_angle=None):
             self.sin = np.sin(iq_angle)
 
     def bake(self, config):
```

Comment:
It might be helpful to add a type hint here:
```suggestion
    def bake(self, config: QMConfig):
```

But since type hints are evaluated right after parsing the line, you should swap `QMConfig` above this object (until at least [PEP 563](https://peps.python.org/pep-0563/) or similar).

Reply:
Done.